build/docker/minio/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# ----------------------------------------------------------------------------
+# (C) Copyright IBM Corp. 2019, 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+# ----------------------------------------------------------------------------
+
+FROM minio/minio
+
+# Indicate that we expect to connect to the minio service on port 8443
```

Comment:
```suggestion
# Indicate that we expect to connect to the minio service on port 9000
```

Reply:
good catch, forgot to change this accordingly